src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { PreparedTodos } from '../Todolist/PreparedTodos';
+
+type Props = {
+  todo: PreparedTodos,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <div className="TodoInfo">
+      <input
+        type="checkbox"
+        id="todo.id"
+        defaultChecked={todo.completed}
+        className="TodoInfo__toggler"
+      />
+
+      <label htmlFor="todo.id" className="TodoInfo__title">
```

Comment:
```suggestion
        id={todo.id}
        defaultChecked={todo.completed}
        className="TodoInfo__toggler"
      />

      <label htmlFor={todo.id} className="TodoInfo__title">
```

now on your page clickinng on any label toggles only first chekcbox